Tests/LibWeb/Text/expected/wpt-import/css/css-fonts/parsing/font-variant-valid.txt

```diff
@@ -49,4 +49,4 @@ Pass	e.style['font-variant'] = "ruby" should set the property value
 Pass	e.style['font-variant'] = "sub" should set the property value
 Pass	e.style['font-variant'] = "super" should set the property value
 Fail	e.style['font-variant'] = "common-ligatures discretionary-ligatures historical-ligatures contextual small-caps stylistic(flowing) lining-nums proportional-nums diagonal-fractions ordinal slashed-zero jis78 full-width ruby sub" should set the property value
-Pass	e.style['font-variant'] = "super proportional-width jis83 stacked-fractions tabular-nums oldstyle-nums historical-forms all-small-caps no-contextual no-historical-ligatures no-discretionary-ligatures no-common-ligatures" should set the property value
\ No newline at end of file
+Fail	e.style['font-variant'] = "super proportional-width jis83 stacked-fractions tabular-nums oldstyle-nums historical-forms all-small-caps no-contextual no-historical-ligatures no-discretionary-ligatures no-common-ligatures" should set the property value
```

Comment:
Curious why this regressed, unintentionally I guess?

Reply:
It's sensitive to the exact order. The test checks for two valid serializations, which confuses me a bit because the general rule is there should be 1, and it should match the order in the grammar. This parsing-change commit made it fail, but the serialization-change commit that follows made it pass again. It was quite confusing working on it. :sweat_smile: